### ISSUE TITLE
The ability to provide custom render instructions.

### ISSUE THREAD

**Issue by KieronWiltshire** [surgeplay/Visage#2]
I think it'd be a great idea to be able to pass custom instructions through to the service.

**An example:**
If I wanted to have an avatar with his fist in the air whilst jumping (jump for joy), I could pass in some instructions that would say, how to render the leg (think of how Cinema4D and  3DS Max rigs work).

**Comment by exaskye** [comment]
Yeah, I've been intending to add a feature like this, but that'd require adding render sorting. Visage currently has a hard-coded render order that only works for the default pose.

**Comment by KieronWiltshire** [comment]
This would be an amazing feature and definately one I'd use.

**Comment by exaskye** [comment]
<img src="https://unascribed.com/i/3377cb2a.png" width="240px"/>

# :thinking:

**Comment by exaskye** [comment]
As of dfedfa20, this is very possible, now that there's rotation anchor support and the secondary layers don't clobber eachother or the base layers. Going to look into designing a JSON format for defining limb/head positions and deploying it next time I work on Visage.

**Comment by KieronWiltshire** [comment]
Epic work man, really appreciate this!
